components/script/dom/element.rs

```diff
@@ -1190,23 +1190,35 @@ impl Element {
 
     // Element branch of https://dom.spec.whatwg.org/#locate-a-namespace
     pub fn locate_namespace(&self, prefix: Option<DOMString>) -> Namespace {
-        let prefix = prefix.map(String::from).map(LocalName::from);
+        let ns_prefix = prefix.clone().map(|s| Prefix::from(&*s));
+
+        // Step 1.
```

Comment:
Would it be possible to copy the text from the specification here. The step numbers often change and the spec language will give a clue as to the new numbers.